src/plugins/intel_cpu/tests/functional/shared_tests_instances/low_precision_transformations/groupconvolution_qdq_transformation.cpp

```diff
@@ -301,7 +301,7 @@ const std::vector<LayerTestsDefinitions::GroupConvolutionQDqTransformationParam>
         {},
         {
             { ngraph::element::f32, false },
-            { {128.f}, ngraph::element::f32, {}, false, 1ul, ngraph::element::i8, true },
+            { {127.f}, ngraph::element::f32, {}, false, 1ul, ngraph::element::i8, true },
```

Comment:
Can we change precision here (not value) to be aligned with the comment above?
```suggestion
            { {128.f}, ngraph::element::f32, {}, false, 1ul, ngraph::element::u8, true },
```

Reply:
I tried this change but after that test cannot find output layer.

Reply:
I reproduced the behavior that you had mentioned. In this case, comment, not test-case parameter should be corrected

Reply:
Fixed